pages/governance.md

```diff
@@ -0,0 +1,110 @@
+---
+title:  "Governance"
+---
+
+This document contains the formal goverance structure of the JuMP project. It
+clarifies how decisions are made with respect to community interactions,
+including the relationship between open source development and work that may be
+funded by for-profit and non-profit entities.
+
+## Code of Conduct
+
+The JuMP community strongly values inclusivity and diversity. Everyone should
+treat others with the utmost respect. Everyone in the community must adhere to
+the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct) and the
+[Julia Community Standards](https://julialang.org/community/standards/), which
+reflect the values of our community. Both these links have information on how to
+confidentially report a violation.
+
+## Steering Council
+
+JuMP is a [fiscally sponsored project of NumFOCUS](https://numfocus.org/project/jump),
+a 501(c)(3) public charity in the United States. NumFOCUS help the JuMP project
+by providing legal and financial support.
+
+JuMP has a Steering Council that acts as the interface between the community and
+NumFOCUS. The main role of the Steering Council is to:
+
+* Approve expenditures related to JuMP, e.g., the renewal of our [jump.dev](https://jump.dev)
+  domain name.
+* Negotiate and approve contracts between NumFOCUS and external contractors who
+  provide paid work to the JuMP project.
+* Represent JuMP in interactions with NumFOCUS, e.g., by attending NumFOCUS's
+  annual summit.
+
+The current members of the steering committee are:
+
+* [Miles Lubin (Google)](https://github.com/mlubin)
+* [Juan Pablo Vielma (Google)](https://github.com/juan-pablo-vielma)
+* [Joey Huchette (Rice University)](https://github.com/joehuchette)
+* [Oscar Dowson](https://github.com/odow)
+* [Changhyun Kwon (U. South Florida)](https://github.com/chkwon)
+
+If a member of the steering committee leaves the committee, the remaining
+members will invite a member of the community to join in order to maintain a
+quorum of five members.
+
+### Conflict of interest
+
+It is expected that Steering Council members will be employed at a wide range of
+companies, universities and non-profit organizations. Because of this, it is
+possible that Members will have conflicts of interest. Such conflicts of
+interest include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work,
+  outside of the Project that may influence their work on the Project.
+* Access to proprietary information of their employer that could potentially
+  leak into their work with the Project.
+
+All members of the Council shall disclose any conflict of interest they may
```

Comment:
shall disclose to whom? to the public or to the other members of the council?

Reply:
Just to the council?